public class Hd {

    public static void main(String [] args) {
        String url = "https://www.google.com";
        go(url);
    }

    private static void go(String url) {
        try {
            System.out.println(new URL(url));
        } catch (MalformedURLException e) {
            e.printStackTrace();
        }
    }

}
```

Here, the `go` method catches the `MalformedURLException` and simply prints the stack trace. This is more appropriate in some cases where the program can continue to run despite the error. However, in many cases it may be best to let the exception be thrown so that it can be handled in a higher level of the program.
